post_at as option in schedule message has the wrong type

### Description
I think `ChatScheduleMessageArguments` interface has wrong type for `post_at` property. In my opinion, it should be `number` instead of `string`.

### What type of issue is this?
- [x] bug
- [ ] enhancement (feature request)
- [ ] question
- [ ] documentation related
- [ ] testing related
- [ ] discussion

### Requirements
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

---

### Bug Report

As in [documentation in "code"](https://api.slack.com/methods/chat.scheduleMessage/code) section we see `post_at` as number. While `@slack/web-api` package tells me that property should be a string.
As I try to call `webClient.chat.scheduleMessage({..., post_at: <Unix Epoch timestamp as string>})` I encountered a problem in response:
```
[ERROR]  web-api:WebClient:0 must provide a number [json-pointer:/post_at]
error Error: An API error occurred: invalid_arguments
    at Object.platformErrorFromResult (C:\<project-path>\node_modules\@slack\web-api\src\errors.ts:94:5)
    at WebClient.apiCall (C:\<project-path>\node_modules\@slack\web-api\src\WebClient.ts:196:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
{
  code: 'slack_webapi_platform_error',
  data: {
    ok: false,
    error: 'invalid_arguments',
    response_metadata: { messages: [Array], scopes: [Array], acceptedScopes: [Array] }
  }
}
```

#### Packages:

Select all that apply:

- [x] `@slack/web-api`
- [ ] `@slack/events-api`
- [ ] `@slack/interactive-messages`
- [ ] `@slack/rtm-api`
- [ ] `@slack/webhooks`
- [ ] `@slack/oauth`
- [ ] `@slack/socket-mode`
- [ ] I don't know

#### Reproducible in:

package version: 6.2.3

node version: 12.18.3

OS version(s): Windows 10

Hi @Zummek, thanks a lot for taking the time to point the issue out. You're right that the recommended data type for `post_at` is `number`, not `string`. I just submitted a pull request #1242  fixing the issue. The fix will be included in the next release.

One thing I noticed is that actually `string` also works for `post_at` argument as long as the value is a valid integer value. I'm wondering if your input value might not be an integer value. Can you check your input data again?

```js
// This value '1621497501.644' will be rejected by the Slack server-side.
// {"ok":false,"error":"invalid_arguments","response_metadata":{"messages":["[ERROR] must provide a number [json-pointer:/post_at]"]}}
const postAt = "" + (Date.now() / 1000) + 60 * 15;

// This value  '1621497501' will be accepted by the Slack server-side.
const postAt = "" + Number.parseInt((Date.now() / 1000) + 60 * 15);
```

--- a/packages/web-api/src/methods.ts
+++ b/packages/web-api/src/methods.ts
@@ -1225,7 +1225,7 @@ export interface ChatPostMessageArguments extends WebAPICallOptions, TokenOverri
 export interface ChatScheduleMessageArguments extends WebAPICallOptions, TokenOverridable {
   channel: string;
   text?: string;
-  post_at: string;
+  post_at: string | number;
   as_user?: boolean;
   attachments?: MessageAttachment[];
   blocks?: (KnownBlock | Block)[];

--- a/packages/web-api/test/types/webclient-named-method-types.test-d.ts
+++ b/packages/web-api/test/types/webclient-named-method-types.test-d.ts
@@ -65,7 +65,13 @@ expectType<Promise<ChatScheduleMessageResponse>>(web.chat.scheduleMessage({
 }));
 expectType<Promise<ChatScheduleMessageResponse>>(web.chat.scheduleMessage({
   channel: 'C111',
-  post_at: '11111',
+  post_at: '1621497568',
+  text: 'Hi there!',
+  blocks: [],
+}));
+expectType<Promise<ChatScheduleMessageResponse>>(web.chat.scheduleMessage({
+  channel: 'C111',
+  post_at: 1621497568,
   text: 'Hi there!',
   blocks: [],
 }));

--- a/prod-server-integration-tests/test/web-api.js
+++ b/prod-server-integration-tests/test/web-api.js
@@ -28,4 +28,30 @@ describe('Web APIs', function () {
       assert.isUndefined(response.error);
     });
   });
+
+  describe('cha.scheduleMessage', function () {
+    it('should accept either an integer or a string value for post_at', async function() {
+      const channelId = process.env.SLACK_SDK_TEST_WEB_TEST_CHANNEL_ID;
+      const postAt = Number.parseInt((Date.now() / 1000) + 60 * 15);
+      try {
+        const params = {
+          text: 'Hi there!',
+          channel: channelId,
+          post_at: postAt,
+        };
+        assert.isTrue(typeof params.post_at === 'number');
+        assert.isTrue(Number.isInteger(params.post_at));
+        const response1 = await botClient.chat.scheduleMessage(params);
+        assert.isUndefined(response1.error);
+
+        params.post_at = "" + params.post_at;
+        assert.isTrue(typeof params.post_at === 'string');
+        const response2 = await botClient.chat.scheduleMessage(params);
+        assert.isUndefined(response2.error);
+      } catch (e) {
+        console.log(e.code + " / " + JSON.stringify(e.data));
+        throw e;
+      }
+    });
+  });
 });

